### ISSUE TITLE
Trigger checkout.session.completed throwing Trigger failed 400

### ISSUE THREAD

**Issue by renjithspace** [stripe/stripe-cli#367]
### Issue
I receiving the following error when I run `stripe trigger checkout.session.completed`

```
  "error": {
    "message": "As per Indian regulations, export transactions require a customer name and address. More info here: https://stripe.com/docs/india-exports",
    "payment_intent": {
 ...
```

### Expected Behavior
Get complete the trigger without any error

### Steps to reproduce
```
$ stripe trigger checkout.session.completed
```

### Traceback
```
$ stripe trigger checkout.session.completed

stripe trigger checkout.session.completed
Setting up fixture for: checkout_session
Setting up fixture for: payment_page
Setting up fixture for: payment_method
Setting up fixture for: payment_page_confirm
Trigger failed: Request failed, status=400, body={
  "error": {
    "message": "As per Indian regulations, export transactions require a customer name and address. More info here: https://stripe.com/docs/india-exports",
    "payment_intent": {
 ...
```

### Environment
macOS

**Comment by ob-stripe** [comment]
Hi @renjithspace, thanks for the report! We just released v1.3.2 which should fix this issue.

**Comment by renjithspace** [comment]
@ob-stripe Thanks!
